### PR TITLE
add probability bounty explanation

### PR DESCRIPTION
**Pull request by pblivin0x**
intuitive explanation, can add more color if desired.

### CODE REVIEW

**Comment by pblivin0x** [comment]
Each batch mint has an isomorphic relation to offset by random number. This isomorphic relation can be achieved by renumbering open slots.

**Comment by 0xngmi** [comment]
why 10e4?

**Comment by 0xngmi** [comment]
10e4 = 100000
10^4 = 10000

**Comment by 0xngmi** [comment]
sorry I don't understand your explanation, could you elaborate on it?

**Comment by pblivin0x** [comment]
> sorry I don't understand your explanation, could you elaborate on it?

yes! we can 1 to 1 map the procedure of each batch problem to a distinct version of offset by random number with (10e4 - nBatch * 10e3).
